### DISCUSSION TITLE
Lesson 1: How do I get keyword shortcuts to appear on remix ide?

### DISCUSSION THREAD

**Discussion by ADFQuinna** [smartcontractkit/full-blockchain-solidity-course-py#1426] [Q&A]
Like when I type "in" it will show a drop down box of different functions that start with "in" (like int or int256) then allow me to click or tab to auto-complete. 

Its not working for me somehow, i have to write anything manuel

**Comment by cromewar** [comment]
Hello @ADFQuinna 
At first when you open REMIX you should add the solidity plugins 
![image](https://user-images.githubusercontent.com/20199180/164052036-efe1baf6-a1ee-42c9-9258-20c82eb89c01.png)
Also you can try using another browser, like Firefox.
